deploy/cr.yaml

```diff
@@ -11,6 +11,7 @@ metadata:
 #    percona.com/issue-vault-token: "true"
 spec:
   crVersion: 1.14.0
```

Comment:
Should we have 1.14.1?

Reply:
doesn't matter but yes, I'll update after pushing 1.15.1